src/test/scala/com/fulcrumgenomics/util/SequencesTest.scala

```diff
@@ -48,25 +49,46 @@ class SequencesTest extends UnitSpec {
     Sequences.countMismatches("ACGACTATATGCAT", "NNNNNNNNNNNNNN") shouldBe 14
   }
 
-  "Sequences.longestHomopolymer" should "fail when passed a null String" in {
-    an[Throwable] shouldBe thrownBy { Sequences.longestHomopolymer(null) }
+  "Sequences.longestHomopolymer" should "fail when passed a zero length string" in {
+    an[Throwable] shouldBe thrownBy { Sequences.longestHomopolymer("") }
   }
 
-  it should "fail when passed a zero length string" in {
-    an[Throwable] shouldBe thrownBy { Sequences.longestHomopolymer("") }
+  "Sequences.maxPolyX" should "fail when passed a null String" in {
+    an[Throwable] shouldBe thrownBy { Sequences.maxPolyX(null) }
+  }
+
+  it should "return zero when passed a zero length string" in {
+    Sequences.maxPolyX("") shouldBe OffsetAndLength(0, 0)
   }
 
   it should "correctly find the single longest homopolymer" in {
-    Sequences.longestHomopolymer("A") shouldBe (0,1)
-    Sequences.longestHomopolymer("AAAA") shouldBe (0,4)
-    Sequences.longestHomopolymer("ATTTGCGAT") shouldBe (1,3)
-    Sequences.longestHomopolymer("GGGGCGTGC") shouldBe (0,4)
-    Sequences.longestHomopolymer("ACGTACGTCCCCC") shouldBe (8,5)
-    Sequences.longestHomopolymer("AAAACCCGGGGAAAAA") shouldBe (11,5)
+    Sequences.maxPolyX("A") shouldBe OffsetAndLength(0,1)
+    Sequences.maxPolyX("AAAA") shouldBe OffsetAndLength(0,4)
+    Sequences.maxPolyX("ATTTGCGAT") shouldBe OffsetAndLength(1,3)
+    Sequences.maxPolyX("GGGGCGTGC") shouldBe OffsetAndLength(0,4)
+    Sequences.maxPolyX("ACGTACGTCCCCC") shouldBe OffsetAndLength(8,5)
+    Sequences.maxPolyX("AAAACCCGGGGAAAAA") shouldBe OffsetAndLength(11,5)
+    Sequences.maxPolyX("ATTCCCGG") shouldBe OffsetAndLength(3,3)
   }
 
   it should "report the earliest homopolymer of the longest length when there are multiple" in {
-    Sequences.longestHomopolymer("AAAAACCCCCGGGGGTTTTT") shouldBe (0,5)
-    Sequences.longestHomopolymer("ACGTAACCGGTTAAACCCGGGTTT") shouldBe (12,3)
+    Sequences.maxPolyX("AAAAACCCCCGGGGGTTTTT") shouldBe OffsetAndLength(0,5)
+    Sequences.maxPolyX("ACGTAACCGGTTAAACCCGGGTTT") shouldBe OffsetAndLength(12,3)
+  }
+
+  "Sequences.maxDinuc" should "correctly count the longest dinucleotide in sequences" in {
+    Sequences.maxDinuc("A")        shouldBe OffsetAndLength(0, 0)
+    Sequences.maxDinuc("ACGT")     shouldBe OffsetAndLength(0, 2)
+    Sequences.maxDinuc("ACACGTT")  shouldBe OffsetAndLength(0, 4)
+    Sequences.maxDinuc("AGTGTGT")  shouldBe OffsetAndLength(1, 6)
+    Sequences.maxDinuc("GGGGGGGG") shouldBe OffsetAndLength(0, 8)
+    Sequences.maxDinuc("AGCGtagCGCGCgcGCTCTCTatCGCGCA") shouldBe OffsetAndLength(6, 10)
+  }
+
+  "Sequences.complement" should "return the complement of sequences" in {
+    Sequences.complement("AAA") shouldBe "TTT"
+    Sequences.complement("ACAC") shouldBe "TGTG"
+    Sequences.complement("") shouldBe ""
+    Sequences.complement("AACCGGTGTG") shouldBe "TTGGCCACAC"
   }
```

Comment:
I know it's only a call to a htsjdk function, but could you add a few basic tests for `reverseComplement()` please?  That way if the library function gets broken/changed unexpectedly we'll catch it, or if we decide to reimplement we won't be caught short.

Reply:
Done.